spec/models/abilities_spec.rb

```diff
@@ -4,105 +4,76 @@
 
 RSpec.describe Mongoidable::Abilities do
   describe "to_list" do
+    let(:expected_result) do
+      [
+          {
+              type:        :adhoc,
+              source:      "test",
+              has_block:   false,
+              subject:     [:to_thing],
+              actions:     [:do_thing],
```

Comment:
Casl is raising a deprecation warning on this key.  It's been changed to just `action`.